src/main/java/com/autotune/analyzer/serviceObjects/BulkJobStatus.java

```diff
@@ -115,130 +108,310 @@ public void setExperiments(Map<String, Experiment> experiments) {
         this.experiments = experiments;
     }
 
-    public Webhook getWebhook() {
-        return webhook;
+    public void copyByPattern(String regex) {
```

Comment:
Can you add a Javadoc for this method, briefly explaining its purpose and functionality?

Reply:
done